src/Payload.php

```diff
@@ -302,14 +309,36 @@ public function getCustomValue($key)
         return $this->customValues[$key];
     }
 
+    /**
+     * Set JSON Options.
+     *
+     * @param $options
+     * @return mixed
+     */
+    public function setJSONOptions($options)
+    {
+        $this->jsonOptions = $options;
+        return $this;
+    }
+
+    /**
+     * Get JSON Options.
+     *
+     * @return Integer
+     */
+    public function getJSONOptions()
+    {
+        return $this->jsonOptions;
+    }
+
     /**
      * Convert Payload to JSON.
      *
      * @return string
      */
     public function toJson(): string
     {
-        return json_encode($this, JSON_UNESCAPED_UNICODE | JSON_FORCE_OBJECT);
+        return json_encode($this, $this->jsonOptions);
```

Comment:
I came up with another option.
We can remove JSON_FORCE_OBJECT if `$this` is not empty, and use JSON_FORCE_OBJECT when `$this` is empty. And we always will have an object as a result.

Reply:
Yeah it would have worked but quite it feels like a hack. Correcting the type of the APS key was a better solution IMO.